examples/http3-client.rs

```diff
@@ -71,9 +72,31 @@ fn main() {
     let poll = mio::Poll::new().unwrap();
     let mut events = mio::Events::with_capacity(1024);
 
+    // Take a look at server address resolved to check if it's ipv4 or ipv6.
+    // Depending on the IP family, bind_addr will be default address of
+    // v4 or v6 for calling bind() later.
+    // This workaround is to work with MacOS (or BSD variants which
+    // doesn't allow v4 and v6 can be bind() in one socket).
+    // Note that linux doesn't need this because it can handle v4 and v6 socket
+    // when bind() with "::".
+    let mut addrs_iter = url.to_socket_addrs().unwrap();
+    let addr = addrs_iter.next(); // get next IP address resolved from URL hostname
+    let bind_addr;
+    match addr {
+        Some(ip) => {
+            debug!("addr = {:?}", ip);
```

Comment:
Something that is a little annoying with http3-client is that it doesn't give much feedback on what is happening when no RUST_LOG var is set. If we are going to add some logging anyway, maybe we should consider to print!() a string that states "Making request to <URL>, resolved to IP".

Reply:
Yes we need to add a command line option somthing like `-v` and add more curl-like options for storing body into file, which can help easier troubleshooting. Let me work on another PR.